spec/implementation_matrix.md

```diff
@@ -11,7 +11,7 @@
 | ethereumjs   |      |      |      |      |      |      |      |      |      |      |       |
 | evmone       | :+1: | :+1: | :+1: | :+1: | :+1: | :+1: | :+1: | :+1: | :+1: | 🚧    | :+1:  |
 | geth         | 🚧   | 🚧    | 🚧   | 🚧   | 🚧    | 🚧   | 🚧   | 🚧    | 🚧   |      | 🚧     |
-| nethermind   |      |      |      |      |      |      |      |      |      |      |       |
+| nethermind   | 🚧   | 🚧    | 🚧   | 🚧   | 🚧    | 🚧   | 🚧   | 🚧    | 🚧   |      | 🚧     |
```

Comment:
If we follow Besu, we could consider this `:+1:` too.

Reply:
Given it is me and not Nethermind updating this, the wip icon makes more sense.

Reply:
Maybe we can summon someone from Nethermind to confirm or amend this? I don't know any appropriate handle though

Reply:
I think they can update later if there's a disagreement. I believe the blocker was mostly getting the data in, people are mostly okay updating/fixing stuff.